app/policies/podcast_ownership_policy.rb

```diff
@@ -0,0 +1,27 @@
+class PodcastOwnershipPolicy < ApplicationPolicy
+  def update?
+    user_is_owner? || user_admin?
+  end
+
+  def new?
+    true
```

Comment:
probably should be the same as `create?`, forbidding this action to banned users